docs/guides/Orders/orders-feed.md

```diff
@@ -260,49 +237,38 @@ Here are two complete example bodies for the [Feed configuration response](https
+  "body": "When a new feed is configured, its queue contains whatever orders are changed right after the setup is complete. If the feed is reconfigured, events from the former queue will remain in the feed until they are committed or until the retention period expires."
 }
 [/block]
-
->❗ Should the Feed not receive any new events in its queue for the time set in `messageRetentionPeriodInSeconds`, your configuration will be removed, and you will have to reconfigure it, using the [Feed configuration API call](https://developers.vtex.com/vtex-rest-api/reference/feed-v3#feedconfiguration) in order to continue to use the Feed. So it is important to be mindful of the filter configuration you are using. You can check it at any time using the [Get feed configuration](https://developers.vtex.com/vtex-rest-api/reference/feed-v3#getfeedconfiguration) endpoint.
+> ❗ If the feed doesn't receive any new events in its queue during the time set in `messageRetentionPeriodInSeconds`, your configuration will be removed, and you will have to reconfigure it with the [Feed configuration API call](https://developers.vtex.com/docs/api-reference/orders-api#get-/api/orders/feed/config) to continue using the feed. Therefore, it's important to be mindful of the filter configuration you are using. You can check it any time using the [Get feed configuration](https://developers.vtex.com/docs/api-reference/orders-api#get-/api/orders/feed/config) endpoint.
 
 ## Feed readout
 
-Every system that depends on order updates should consume the Feed to be able to take the necessary actions based on that information. The most common behavior is, therefore, that of a store system or an integration reading every event in the Feed and, based on the status, making a decision for each one.
-[block:callout]
-{
-  "type": "warning",
-  "body": "When filtering statuses, be aware that all possible order statuses must be dealt with during integration to avoid errors. Special attention should be given to `Status Null`, which may be unidentified and end up being mapped as another status, potentially leading to errors."
-}
-[/block]
+Every system that depends on order updates should consume the feed to be able to take the necessary actions based on that information. The most common behavior is a store system or an integration reading every event in the feed and, based on its status, making a decision for each one. [block:callout]
+{ "type": "warning", "body": "When filtering statuses, be aware that all possible order statuses must be dealt with during integration to avoid errors. Particular attention should be paid to `Status Null`, which may be unidentified and end up being mapped as another status, which can potentially lead to errors." } [/block]
```

Comment:
```suggestion
>⚠️ When filtering statuses, be aware that all possible order statuses must be dealt with during integration to avoid errors. Particular attention should be paid to `Status Null`, which may be unidentified and end up being mapped as another status, which can potentially lead to errors.
```